Subject: Turn off EARLY_PRINTK on IP22 to make the system boot

Thomas Bogendoerfer: current kernels are not working on that machine
due to some bad interaction between CONFIG_EARLY_PRINTK and
overwritten prom memory during early boot (this bug is on my
todo list.

In the meantime, turn off CONFIG_EARLY_PRINTK on IP22.

Debian bug #507557

--- a/arch/mips/Kconfig	2009-09-22 09:22:46.000000000 +0000
+++ b/arch/mips/Kconfig	2009-09-22 09:22:59.000000000 +0000
@@ -357,7 +357,6 @@
 	select SWAP_IO_SPACE
 	select SYS_HAS_CPU_R4X00
 	select SYS_HAS_CPU_R5000
-	select SYS_HAS_EARLY_PRINTK
 	select SYS_SUPPORTS_32BIT_KERNEL
 	select SYS_SUPPORTS_64BIT_KERNEL
 	select SYS_SUPPORTS_BIG_ENDIAN

